pic-sure-auth-services/src/main/java/edu/harvard/hms/dbmi/avillach/auth/service/TermsOfServiceService.java

```diff
@@ -56,7 +57,8 @@ public void acceptTermsOfService(String userId){
         }
         user.setAcceptedTOS(new Date());
         List<User> users = Arrays.asList(user);
+        Date tosDate = termsOfServiceRepo.getLatest().getDateUpdated();
         userService.updateUser(users);
+        logger.info("User " + (!StringUtils.isEmpty(user.getEmail()) ? user.getEmail() : user.getGeneralMetadata()) + " accepted the Terms of Service dated " + tosDate.toString());
```

Comment:
This log statement is missing the prescribed sentinel included in the ticket's description. This is used to filter messages for the Splunk dashboard.

TOS_LOG :